src/Board.svelte

```diff
@@ -243,6 +277,29 @@
       transition:fade={{ duration: 200 }}
       class="d-none d-lg-block scroll h-100"
     >
+      {#if showIceBreaking}
```

Comment:
Can we put this section into a new component that contains all the edit and visibility code?
Then we can just have something like `<IceBreaker>` here and it'll manage itself.

Reply:
Done!